test/helper/helper.go

```diff
@@ -312,21 +315,22 @@ func CleanupProjectAndDevice(deviceId, projectId string) error {
 //nolint:staticcheck
 func SetupProjectAndDevice(t *testing.T, projectId, deviceId *string) *openapiclient.Device {
 	t.Helper()
-	projId, err := CreateTestProject("metal-cli-test-project")
+	projectName := "metal-cli-test-project" + GenerateUUID()
```

Comment:
The project name prefix here should be passed in, not hard-coded to `metal-cli-test-project`; I see a couple orphaned projects from these tests, but I can't tell which tests they came from because of the generic prefix.

Reply:
Yes right. Fixed